src/lib/Microsoft.Health.Events/Model/EventMessage.cs

```diff
@@ -31,7 +32,7 @@ public EventMessage(string partitionId, DateTime dateTime)
             Offset = offset;
             EnqueuedTime = enqueuedTime;
             Properties = new Dictionary<string, object>(properties);
-            SystemProperties = new Dictionary<string, object>(systemProperties);
+            SystemProperties = systemProperties.ToDictionary(kv => kv.Key, kv => kv.Value);
```

Comment:
This might result in different behavior if the passed in dictionary is null.  Can we verify that an empty SystemProperties is created if the passed in value for SystemProperties is null?

Reply:
Creating empty Dictionary if supplied SystemProperties is `null`